docs/FAQ/ServernGamemodes.md

```diff
@@ -0,0 +1,196 @@
+
+> Q: How does hardmode die?
+>
+> A: When the mob cap is reached(if you have [Crew](#Crew) you can do `/worlddata mode` to see the cap and the current kill count).
+
+> Q: What are the warps?
+>
+> A: (As of Oct 20 2019) The warps are Elevator, cavern, lhell, rhell, hell, locean, rocean, dungeon, jungle. You can see a list of warps by doing `/warp list`. locean and rocean are both ocean warps, locean is for the left ocean and rocean is for the right ocean. lhell, hell, rhell are all hell warps, lhell is for left hell, hell is a point in hell close to the center of the world, rhell is for right hell.
+
+> Q: Where can I build a house?
+>
+> A: You can build a house on the spawn platform (the entirety of the asphalt runway), if you build it anywhere else, it will get removed by world regen.
+
+> Q: How can I protect it?
+>
+> A: You can protect it by using the `/house define` command ([Crew](#Crew) is required), after executing the command, it will tell you to hit 2 points in your house, top-left and bottom-right.
+
+> Q: How many houses can I define?
+>
+> A: As of Dec 27 2019, 2 houses can be defined by the player.
+
+> Q: How big can my house be?
+>
+> A: As of Dec 27 2019, houses can occupy an area of 2024 blocks each.
+
+## Others
+
+### Items
+TSL Server, `/items` to join.
+
+Free items dimension.
+
+> Q: I can't find the item I want, how can I spawn it in?
+>
+> A: You can spawn items using the command `/i <item name or id> <stack> <prefix>`.
+	
+### Special Items
+TSL Server, `/specialitems` to join.
+
+Special Items is a dimension where you can modify items, you do not keep the items when you leave the server, the modified items will disappear.
+
+> Q: Why can't I see my friends?
+>
+> A: TSL groups players into groups of 10.
+
+> Q: How can I see other people's projectiles?
+>
+> A: Enable PvP. Caution: they may kill you repeatedly, if this happens, you can do `/rift` to leave to disable pvp and join again.
+
+### Rift
+TSL Server, `/rift` to join.
+
+Rift or formerly Mirror is the dimension you are in when you join the server on port 7777, it is the main hub to all these dimensions. You can't see other players in Rift to reduce lag.
+
+### Gamemodes
+
+TSL Server, `/gamemodes` to join
+
+Gamemodes is a Rift-like dimension, hub for [minigames](#minigames)
+
+
+### Archives
+**Note: Archives are rarely hosted.**
+
+TSL Servers, `/archive0x5`, `/archive0x5b`, `/archive0x2`, `/archive0x3` or `/archive0x4` to join any of the 5 dimensions
+
+Archives are comprised of 5 dimensions `archive0x5`, `archive0x5b`, `archive0x2`, `archive0x3` and `archive0x4`, they are pieces of the Dark Gaming History, they contain the old lobbies of Dark Gaming.
+
+## Closed
+
+### AutoSurvival
+
+TSL Lobby, TShock Worlds, `/autosurvival` to join.
+
+AutoSurvival is a type of survival where the server hosts a survival for you, you can then download the world later on.
```

Comment:
```suggestion
AutoSurvival is a type of survival where the server hosts a survival for you, you can then download the world later on. AutoSurvival only existed to make use of spare server capacity, it may come back in the future if such an event arises.
```